cmd/notifiers.go

```diff
@@ -61,6 +63,7 @@ type notifier struct {
 	Redis         map[string]redisNotify         `json:"redis"`
 	PostgreSQL    map[string]postgreSQLNotify    `json:"postgresql"`
 	Kafka         map[string]kafkaNotify         `json:"kafka"`
+	HTTP          map[string]httpNotify          `json:"http"`
```

Comment:
Thanks for pointing me at that file. I assume from reading the code we'd need a v12 format?